Sources/Honeycomb/MetricKitSubscriber.swift

```diff
@@ -419,10 +419,22 @@
                 attrs["exception.code"] = exceptionCode.intValue
             }
             if let exceptionType = $0.exceptionType {
-                attrs["exception.mach_execution_type"] = exceptionType.intValue
```

Comment:
note this used to be spelled `execution_type` instead of `exception_type`, I assumed that was an error